src/khoj/configure.py

```diff
@@ -303,9 +304,16 @@ def configure_middleware(app):
     app.add_middleware(SessionMiddleware, secret_key=os.environ.get("KHOJ_DJANGO_SECRET_KEY", "!secret"))
 
 
-@schedule.repeat(schedule.every(22).to(26).hours)
+@schedule.repeat(schedule.every(22).to(25).hours)
 def update_search_index():
     try:
+        if ProcessLockAdapters.is_process_locked(ProcessLock.Operation.UPDATE_EMBEDDINGS):
```

Comment:
Should we also add a check here to skip running the job if the Operation succeeded in the last ~24 hours?